Assets/MixedRealityToolkit/Inspectors/MixedRealityPlayspaceInspector.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    [CustomEditor(typeof(MixedRealityPlayspace))]
+    public class MixedRealityPlayspaceInspector : UnityEditor.Editor
+    {
+        private static string description =
+          "This is where the Toolkit instantiates objects like cursors, pointers and boundary visualizations." +
```

Comment:
This feels like non-deterministic behavior from an application stand point (read: an app may not always load scenes in the same order and may get different playspaces each time).

Reply:
I realize we discussed this a bit yesterday in person. It just hit me this morning while re-reviewing.

Reply:
>This feels like non-deterministic behavior

The first playspace you load is treated as the active playspace. Once that's set, any others that are loaded get disabled. So you should be able to load scenes in any order and reliably predict which playspace will be active.

It starts getting weird if you unload the scene with the active playspace. Our workflow recommendations advise against this (keep your manager scene loaded etc) but if you go against that grain, the next active playspace selection will be non-deterministic. We could potentially tie playspace selection to scene build index or something like that, but it's not something I've tried.